hardware_integration/CMakeLists.txt

```diff
@@ -264,6 +271,45 @@ if("TouCAN" IN_LIST CAN_DRIVER)
     )
   endif()
 endif()
+if("SYS_TEC" IN_LIST CAN_DRIVER)
+  if(MSVC)
+    # See https://gitlab.kitware.com/cmake/cmake/-/issues/15170
+    set(CMAKE_SYSTEM_PROCESSOR ${MSVC_CXX_ARCHITECTURE_ID})
+  endif()
+  if(WIN32)
+    if(CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64" OR CMAKE_SYSTEM_PROCESSOR
+                                                   STREQUAL "X86")
+      message(STATUS "Detected x86, linking to SYS TEC x86 library")
+      message(
+        AUTHOR_WARNING
+          "Make sure you've installed the driver for your SYS TEC device before using this plugin.
+          https://www.systec-electronic.com/ ")
+      target_link_libraries(
+        HardwareIntegration
+        PRIVATE ${CMAKE_CURRENT_LIST_DIR}/lib/Windows/Usbcan32.lib)
```

Comment:
Hah, whoops, .gitignore strikes again. Added the .lib files and corrected CMake to have them in all caps, since that's how the files were distributed from SysTec